resources/api/schemas/analytiikka-mhu-toteutuneet-kustannukset-haku-response.schema.json

```diff
@@ -146,6 +146,259 @@
+                "muutoskategoria": {
+                  "type": "string",
+                  "description": "Tavoitehinnan muutoksen luokittelu. Arvot 28.1.2024 (muutoksiin on syytä varautua): Alleviivatun fontin vaikutus tavoitehintaan, Bonukset ja sanktiot, Hoitoluokkamuutokset, Liikennejärjestelyt, Materiaalit, Muut, Tehtävämuutokset, Tiestömuutokset, Tiestömuutos, Työmäärämuutokset, Yleiset Liikennejärjestelyt."
+                },
+                "muutoksen-selite": {
+                  "type": "string",
+                  "description": "Vapaamuotoinen selite tavoitehintaa muuttavalle tiedolle."
+                },
+                "poistettu": {
+                  "type": "boolean",
+                  "default": false,
+                  "description": "True kun muutos on merkitty Harjassa poistetuksi. Oletus: false."
+                }
+              },
+              "required": [
+                "muutos-id",
+                "muutoksen-hoitovuosi",
+                "muutoskategoria",
+                "muutoksen-selite",
+                "muutoksen-maara"
+              ]
+            }
+          }
+        },
+        "hoitovuoden-paatokset": {
+          "type": "array",
+          "properties": {
+            "hoitovuoden-paatos": {
+              "type": "object",
+              "properties": {
+                "paatos-id": {
+                  "type": "integer",
+                  "description": "Hoitovuoden päättämiseen liittyvä rivi Harjan tietokannassa. Hoitovuoden päättyessa pidetään välkatselmus. Tavoitehinnan muutosten ja lupauspisteiden perusteella lasketaan, onko tavoitehinta alittunut tai ylittynyt, onko kattohinta ylittynyt sekä saako urakoitsija lupauksistaan sanktota tai bonusta. Lopputulokset tallentuvat päätösriveinä. Hoitovuoden päättämiseen liittyvät päätösrivit eivät ole omia erillisiä kustannusrivejä. Jos päätösriviin liittyy toteutunut kustannus, siihen viitataan viittaukset-toteutuneisiin-kustannuksiin-osion id:llä. Toteutunut kustannus voi olla negatiivinen tai positiivinen kulu, sanktio tai bonus. Hoitokauden päättämiseen liittyvästä päätösrivistä ei aina generoidu toteutunutta kustannusta. Aina kynnys kustannuksen generoimiseen ei ylity. Rivi voi sisältää myös seuraavalle hoitokaudelle kustannussuunnitelmaan siirtyvän negatiivisen tai positiivisen summan. Rivin sisältöä kuvaa paatostyyppi-attribuutti."
+                },
+                "paatoksen-hoitovuosi": {
+                  "type": "string",
+                  "format": "date",
+                  "description": "Sen hoitovuoden aloitusvuosi, jota päätös koskee. Jos kyseessä on hoitovuosi 2023-24, tässä palautuu 2023."
+                },
+                "paatostyyppi": {
+                  "type": "string",
+                  "description": "Hoitovuoden päätökseen liityvän rivin sisältöä kuvaava tyyppi. Harjan tietokannassa enumeraatio: 'tavoitehinnan-ylitys', 'kattohinnan-ylitys', 'tavoitehinnan-alitus', 'lupausbonus', 'lupaussanktio'"
```

Comment:
Näitä tulee välikatselmuksen uudistuessa lisää seuraavan 6kk aikana. Siitä voisi ehkä varoittaa tässä.